src/main/java/core/basesyntax/reportservice/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.reportservice;
+
+import core.basesyntax.storage.DateFruits;
+import java.util.stream.Collectors;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    @Override
+    public String getReport() {
+        return "fruit,quantity" + "\n" + DateFruits.getAll().entrySet().stream()
+                .map(element -> element.getKey() + "," + element.getValue())
+                .collect(Collectors.joining("\n"));
```

Comment:
Create constants for literals. Use System.lineseparator()
What is bad with this type of concatenation?